tests/x509/test_x509_ext.py

```diff
@@ -5769,6 +5769,15 @@ def test_public_bytes(self):
 
 
 class TestSignedCertificateTimestamps:
+    def test_strip_scts(self, backend):
+        cert = _load_cert(
+            os.path.join("x509", "cryptography-scts.pem"),
+            x509.load_pem_x509_certificate,
+            backend,
+        )
+
+        assert cert.tbs_certificate_bytes != cert.tbs_precertificate_bytes
```

Comment:
Flagging for review: this "test" really only checks that we return something different than the normal `tbsCertificate` body, since we know that this cert has an SCT list extension.

Ideally we'd do something more behavioral like refute the presence of the extension in the `tbsCertificate`, but I don't think `cryptography` has any APIs to load *just* the `tbsCertificate` format outside of its X.509 envelope. Thoughts? Alternatively, I could add a known good hex string here, since we can confirm on `sigstore-python` that this helper is working as expected.

Reply:
Known good value comparison would be preferable.

Reply:
Sounds good. @tetsuo-cpp, do you happen to have any of the outputs you produced from experimenting with `certificate-transparency-go`? Those would probably make for a good testvector candidates.

Reply:
Yep, I can do this.

Reply:
Done, and documented.